### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by edellucien**
# How to

Launch the preview and test it.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

The following updates will be done in antoher PR

- [x] Update Cypress
- [ ] ~~Update React~~
- [x] Optional Update Matomo
- [ ] ~~Optional Update React-styleguidist~~
- [x] Optional Update Husky

### CODE REVIEW

**Review by friedjoff — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -10,42 +10,39 @@
...
     "node-sass": "4.13.1",
     "ol": "6.3.1",
-    "proj4": "2.6.1",
+    "proj4": "2.7.0",
     "prop-types": "15.7.2",
     "proxy-polyfill": "0.3.1",
-    "query-string": "6.13.1",
-    "react": "16.13.1",
+    "query-string": "6.14.1",
+    "react": "16.14.0",
```
Comment (oterral) [inline package.json:30]: react 17 is out

**Review by oterral — CHANGES_REQUESTED** [review]
there is missing update

File: package.json (hunk 1 above)
Comment (edellucien) [reply package.json:32]: Start the work to update to react v17 but it seems not that easy:
https://github.com/geops/trafimage-maps/pull/894

Some test libs are not yet up to date, for example 'enzyme-adapter-react-17' is not officialy published:
https://github.com/enzymejs/enzyme/issues/2429

I would leave that open until react update is doable easily.

Comment (oterral) [reply package.json:32]: I agree

**Comment by oterral** [comment]
I'm pretty sure there is still some missing updates. run `yarn outdated` to see what you can do .

**Review by oterral — APPROVED** [review]

**Pull request merged**